lib/banks/checking.rb

```diff
@@ -0,0 +1,17 @@
+require "pry"
+module BankAccount
+  class CheckingAccount < Account
+      MINIMUM_BALANCE = 0
+      TRANSACTION_FEE = 1
+
+    def withdraw(amount)
+      # This method inherits from the Account class #withdraw(amount) method
+      # amount will be checked against current_balance to make sure you have
+      # enough money in your account to make this withdrawl (leaving at least
+      # $10 left in the account)
```

Comment:
Minor gripe: updating comments after copypasta. `MINIMUM_BALANCE` is zero in this class.